client/me/purchases/billing-history/vat-vendor-details.tsx

```diff
@@ -103,7 +103,7 @@ export function VatVendorDetails( { transaction }: { transaction: BillingTransac
 
 	return (
 		<li>
-			<strong>{ translate( 'Vendor VAT Details' ) }</strong>
+			<strong>{ translate( 'Vendor ' ) + vendorInfo.taxName + translate( ' Details' ) }</strong>
```

Comment:
Same advice here: we should only use one translation string, but it can contain a variable. For a similar example, see https://github.com/Automattic/wp-calypso/pull/74538